helm/trento-ai-companion/questions.yml

```diff
@@ -0,0 +1,221 @@
+# Copyright 2025 SUSE LLC
+# SPDX-License-Identifier: Apache-2.0
+
```

Comment:
This file is used by Rancher to provide a user-friendly deployment view. We can also autogenerate it quite easily. Maybe it is not the preferred deployed option, but sharing it just in case.